src/Protocol/Diaspora.php

```diff
@@ -2810,6 +2810,26 @@ private static function isSolicitedMessage(string $url, int $uriid, string $auth
 		return Relay::isSolicitedPost($tags, $body, $contact['id'], $url, Protocol::DIASPORA);
 	}
 
+	/**
+	 * Store an attached photo in the post-media table
+	 *
+	 * @param int $uriid
+	 * @param object $photo
+	 * @return void
+	 */
+	private static function storePhoto(int $uriid, $photo)
```

Comment:
Please rename to `storePhotoAsMedia`.
```suggestion
	private static function storePhotoAsMedia(int $uriid, $photo)
```

Reply:
Done